front/itilsolution.form.php

```diff
@@ -43,6 +43,7 @@
 Session::checkLoginUser();
 
 $solution = new ITILSolution();
+/** @var CommonITILObject $track */
```

Comment:
Apply `getItemForItemtype` changes from main and have a `is_subclass_of` check for the itemtype parameter.